<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PeriodWidget</class>
 <widget class="QGroupBox" name="PeriodWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>110</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Period</string>
  </property>
  <property name="title">
   <string>Period</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="horizontalSpacing">
    <number>3</number>
   </property>
   <property name="margin">
    <number>3</number>
   </property>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Report type:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2" colspan="4">
      <widget class="QComboBox" name="reportTypeCombo">
       <property name="styleSheet">
        <string>QWidget { background:none}</string>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <item>
        <property name="text">
         <string>Daily</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Weekly</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Monthly</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Yearly</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="6">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0" colspan="3">
      <widget class="QComboBox" name="periodCombo">
       <property name="styleSheet">
        <string>QWidget { background:none}</string>
       </property>
       <property name="currentIndex">
        <number>2</number>
       </property>
       <item>
        <property name="text">
         <string>This week</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>This month</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>This year</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Last week</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Last month</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Last year</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Last</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Custom dates</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="3" colspan="2">
      <widget class="QLineEdit" name="numUnitsEdit">
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string>QWidget { background:none}</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5" colspan="2">
      <widget class="QComboBox" name="unitsCombo">
       <property name="styleSheet">
        <string>QWidget { background:none}</string>
       </property>
       <item>
        <property name="text">
         <string>days</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>weeks</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>months</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>years</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="7">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>From:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="3">
      <widget class="QDateEdit" name="fromDate">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string>QWidget { background:none}</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="4" colspan="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>To:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="6" colspan="2">
      <widget class="QDateEdit" name="toDate">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string>QWidget { background:none}</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>1</width>
       <height>80</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
